apps/ui/pages/reservations/[id]/index.tsx

```diff
@@ -467,10 +468,13 @@ function Reservation({
   }, [reservation]);
 
   // NOTE should never end up here (SSR redirect to login)
-  if (!isAuthenticated) {
+  if (!isAuthenticated || reservation.user?.pk !== user?.pk) {
```

Comment:
This is a left over (the next check does it)

Reply:
👍